Octokit/Helpers/AcceptHeaders.cs

```diff
@@ -74,6 +74,8 @@ public static class AcceptHeaders
 
         public const string OAuthApplicationsPreview = "application/vnd.github.doctor-strange-preview+json";
 
+        public const string SearchCommitsPreview = "application/vnd.github.cloak-preview";
```

Comment:
To get the build passing you will need to this rule to the `previewKeys` array in `Octokit.Tests.Conventions/PreviewsTests.cs`

https://github.com/octokit/octokit.net/blob/e41b2d19d74d84e5b5222ebce58d64fc308b1c2e/Octokit.Tests.Conventions/PreviewsTests.cs#L11-L21